save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/HistoryView.kt

```diff
@@ -121,6 +122,8 @@ external interface FiltersProps : TableProps<ExecutionDto> {
     "TOO_MANY_LINES_IN_LAMBDA",
 )
 class HistoryView : AbstractView<HistoryProps, HistoryViewState>(Style.SAVE_LIGHT) {
+    val navigate = useNavigate()
```

Comment:
it can be private, but I'm surprise that it does work.
I thought `useSomething` can be called only inside FC (@akuleshov7 @sanyavertolet )

Reply:
You definitely should not use hooks inside class components